es-vue-base/src/lib-components/EsPopover.vue

```diff
@@ -32,8 +33,9 @@
                 <slot />
                 <div v-if="!hasTitle">
                     <EsButton
+                        :inline="true"
                         variant="link"
-                        class="p-0"
+                        class="p-0 pl-2"
```

Comment:
Per the [design system's spacing page](https://design.energysage.dev/atoms/spacing), `pl-2` is deprecated and should be `pl-50`.